src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,81 @@
+package duke.task;
+
+import duke.util.DukeException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Event extends Task {
+
+    private static final DateTimeFormatter DATE_SHORT_FORMATTER = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    private static final DateTimeFormatter DATE_MED_FORMATTER = DateTimeFormatter.ofLocalizedDate(FormatStyle.LONG);
+
+    private LocalDate on;
+
+    /**
+     * Initialises an Event.
+     *
+     * @param description description of Event
+     * @param on date of Event
+     */
+    public Event(String description, LocalDate on) {
+        super(description);
+        this.on = on;
+    }
+
+    private static void checkFormat(String formattedString) throws DukeException {
+        int onIndex = formattedString.indexOf("/on ");
+        if (onIndex == -1)
+            onIndex = formattedString.length();
+
+        String keyword = formattedString.split(" ", 2)[0];
+
+        if (!keyword.startsWith("event"))
+            throw new DukeException("I can't seem to find the event keyword");
+        else if (formattedString.length() <= 6 || formattedString.substring(6, onIndex).isEmpty())
+            throw new DukeException("the description of event cannot be empty");
+        else if (onIndex == formattedString.length() || formattedString.length() < onIndex + 5)
+            throw new DukeException("the [/on] time of event cannot be empty");
+    }
+
+    /**
+     * Creates an Event given a Event represented as a formatted string.
+     * Format: event [description] /on [DD/MM/YYYY]
+     *
+     * @param formattedString Event represented as a formatted string.
+     * @return Created Event
+     * @throws DukeException given string fails to meet format requirements
+     */
+    public static Event create(String formattedString) throws DukeException {
```

Comment:
I understand what create() does, but could adding  Events at the name like createEvents() make it sound more accurate?